packages/sdk/src/controllers/SubscriberController.ts

```diff
@@ -66,21 +66,44 @@ export class SubscriberController {
         callBack && callBack(unit as MeasurementUnit);
     }
 
+
     /**
-     * Listener on the state of the currently selected frame, if this changes, this listener will get triggered with the updates
-     * @param frameLayout Stringified object of FrameLayoutType
+     * @deprecated use `onSelectedFramesLayoutChanged` instead
+     * Listener on the state of the currently selected frames, if this changes, this listener will get triggered with the updates
+     * @param frameLayout Stringified array of FrameLayoutType objects
```

Comment:
Update the comment to reflect that it will be the first frame when multiple are selected